en/08_Changelogs/6.0.0.md

```diff
@@ -209,12 +211,54 @@ The one change we specifically want to call out is for [`ModelData::obj()`](api:
+
+This does not retroactively affect existing passwords, but will affect any new passwords (e.g. new members or changing the password of an existing member).
+
+If you want to revert to the validator that was used in CMS 5, you can do so with this YAML configuration:
+
+```yml
+---
+After: '#corepasswords'
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Security\Validation\PasswordValidator:
+    class: 'SilverStripe\Security\Validation\RulesPasswordValidator'
+```
+
+See [passwords](/developer_guides/security/secure_coding/#passwords) for more information about password validation.
+
+#### `ConfirmedPasswordField` changes
+
+If [ConfirmedPasswordField->requireStrongPassword](api:SilverStripe\Forms\ConfirmedPasswordField->requireStrongPassword) is set to true, the old behaviour was to validate that at least one digit and one alphanumeric character was included. This meant that you could have a password like "a1" and it would be considered "strong".
+
+This has been changed to use the [`PasswordStrength` constraint](https://symfony.com/doc/current/reference/constraints/PasswordStrength.html) in `symfony/validator` instead. Now a password is considered "strong" based on its level of entropy.
+
+You can change the level of entropy required by passing one of the valid [minScore values](https://symfony.com/doc/current/reference/constraints/PasswordStrength.html#minscore) into [`api:SilverStripe\Forms\ConfirmedPasswordField::setMinPasswordStrength()`](ConfirmedPasswordField::setMinPasswordStrength()).
+
+### Other new features
 
 - Native indexed PHP arrays can now be passed into templates and iterated over with `<% loop $MyArray %>`. Under the hood they are wrapped in [`ArrayList`](api:SilverStripe\Model\List\ArrayList), so you can get the count using `$Count` and use `<% if $ArrayList %>` as a shortcut for `<% if $ArrayList.Count %>`. Other functionality from `ArrayList` such as filtering and sorting cannot be used on arrays since they don't have keys to filter or sort against.
 - Modules no longer need to have a root level `_config.php` or `_config` directory to be recognised as a Silverstripe CMS module. They will now be recognised as a module if they have a `composer.json` file with a `type` of `silverstripe-vendormodule` or `silverstripe-theme`.
 - A new [`DataObject::getCMSEditLink()`](api:SilverStripe\ORM\DataObject::getCMSEditLink()) method has been added, which returns `null` by default. This provides more consistency for that method which has previously been inconsistently applied to various subclasses of `DataObject`. See [managing records](/developer_guides/model/managing_records/#getting-an-edit-link) for more details about providing sane values for this method in your own subclasses.
 - The `CMSEditLink()` method on many `DataObject` subclasses has been renamed to `getCMSEditLink()`.
+- The [`UrlField`](api:SilverStripe\Forms\UrlField) class has some new API for setting which protocols are allowed for valid URLs.
```

Comment:
Can add more details and code examples in its own section if you want. I'm on the fence about doing it so I went for least effort initially.

Reply:
Leave for now, there's a decent chance that the DBField validation work will add some new fields, so we'll see how things look after that